### DISCUSSION TITLE
Triggered data graph - AKA "Logic analyser"

### DISCUSSION THREAD

**Discussion by mengstr** [hneemann/Digital#1106] [General]
I just wanted to say Thank you for the Triggered Data Graph component you added two years ago. I just discovered it and it is so much easier to use compared to the regular Measurement Graph.  The Measurement Graph forced you to do mental gymnastics  when trying to interpret the graph - especially if you're used to a Logic Analyser.  But now it's just like the real thing. Nice!
#571

**Comment by hneemann** [comment]
> I just wanted to say Thank you for the Triggered Data Graph component you added two years ago.

You are welcome! :smile:
